### DISCUSSION TITLE
Add Flags to Route Listing, to enable listing a particular Routes file like php artisan route:list --api-only

### DISCUSSION THREAD

**Discussion by zaghadon** [laravel/framework#43213] [Ideas] [answered]
Can We Add Flags to Route Listing, to enable listing a particular Routes file like `php artisan route:list --api-only` to list only the API Routes, or `php artisan route:list --web-only` and so forth.

First String in the flag should be the same as the routes filename.

**Comment by zaghadon** [comment]
If This doesn't fall in this Repository, Kindly point me to the proper repository and I'll quickly transfer the discussion there.

**Comment by eduance** [comment, marked as answer]
There is already a way to do this; you can use the php artisan route:list --path=api/

**Reply by zaghadon** [reply]
https://github.com/laravel/framework/discussions/43213#discussioncomment-3160855

**Comment by zaghadon** [comment]
I didn't find this in the Documentation. Can we add it on the Docs?

**Reply by eduance** [reply]
You can always consider contributing to the documentation, but it needs to be important enough. You can always, as listed in the documentation, just use the php artisan --help function for more information about it.

Feel free to give it a try: https://github.com/laravel/docs

**Reply by zaghadon** [reply]
Great, I'll go ahead and contribute that to the docs because, most Devs like myself would try to find help on docs first.

Thanks for your replies.

**Comment by zaghadon** [comment]
I have Opened a Pull Request to add this to the Docs here  [#8047](https://github.com/laravel/docs/pull/8047)
